### DISCUSSION TITLE
Error Communication Link Failure

### DISCUSSION THREAD

**Discussion by noorsyamiera92** [dbeaver/dbeaver#13249] [Q&A] [answered]
#### System information: 
- MacOS big Sur 11.4
- DBeaver 21.1.1

#### Connection specification:
- Mysql 5
- dbeaver.io/files/jdbc
- SSH

#### Describe the problem you're observing:

I keep on receiving error 

SQL Error [08S01]: Communications link failure¶¶The last packet successfully received from the server was 351,120 milliseconds ago.  The last packet sent successfully to the server was 42,900 milliseconds ago.

#### Steps to reproduce, if exist:

I able login but after awhile when i want to execute query , it will come out that error.
I need to invalidate it or disconnect which gonna took awhile , then only i able to execute the query.
It will keep on repeating each day.

**Comment by uslss** [comment, marked as answer]
hi @noorsyamiera92 ,
please try to set keep-alive interval in connectionproperties
![image](https://user-images.githubusercontent.com/31996417/123233269-97d5ca00-d4e2-11eb-99a0-3554ef5b475a.png)

**Comment by noorsyamiera92** [comment]
I already put 10000, but its still the same.
I keep on receiving communication link failure.

**Comment by p91paul** [comment]
@noorsyamiera92 you should put it to a lower value, not higher. Like 60.
At 10000 it's probably too late to keep the collection alive.

**Reply by dopamine-fit** [reply]
nothing worked for us as well, any suggestions to solve this?

**Comment by damianbl** [comment]
Same issue with:
Dbeaver backend version: 25.1.5.202508180833
MacOS: 15.6.1 (24G90)
